javascript/preflight.js

```diff
@@ -0,0 +1,46 @@
+function runPreFlightChecks() {
+  const isHttp      = window.location.protocol == 'http:',
+        isLocalhost = window.location.hostname == 'localhost'
+        isUnsafe    = !(isHttp && isLocalhost);
```

Comment:
This seems to have broken fourth-wall for us. We access it using the url: `https://alphagov.github.io/fourth-wall/..`

So this code reads as:
```javascript
const isHttp      = window.location.protocol == 'http:',  // https: == 'http:' => false
      isLocalhost = window.location.hostname == 'localhost' // alphagov.github.io == 'localhost' => false
      isUnsafe    = !(isHttp && isLocalhost); // '!(false && false) => !false => true
```

So even though we are using it over https this is triggering as unsafe. I think the logic you want is:

```javascript
isUnsafe = !http || !(http && isLocalhost)
```
which means if your on https you're safe, or if your on http and localhost your safe.